data/pathfinder/paizo/adventure_path/mummys_mask/mummys_mask_players_guide/mm_abilities.lst

```diff
@@ -1,19 +1,21 @@
+Devotee of the Old Gods		KEY:Trait ~ Devotee of the Old Gods						CATEGORY:Special Ability	TYPE:Trait.CampaignTrait	PREMULT:1,[PREABILITY:1,CATEGORY=Special Ability,Trait ~ Devotee of the Old Gods],[PREVAREQ:BypassTraitRestriction,1],[!PREABILITY:1,CATEGORY=Special Ability,TYPE.CampaignTrait]	DESC:Osirion has a history stretching back over 8,000 years, and the deities worshiped today in modern Osirion are not the same ones revered in Ancient Osirion's heyday-deities with names like Anubis, Osiris, Ra, and Set, among others. Your family never lost the faith of your ancestors, however, and your devotion to one of the deities of Ancient Osirion has helped keep the memory of Osirion's past alive-a past that still lingers on in the untouched necropolis of Wati. You gain a +1 trait bonus on Knowledge (history) and Knowledge (religion) checks, and one of those skills is always a class skill for you. In addition, your faith in the old gods of Osirion grants you a +1 trait bonus on saving throws against divine magic.	STACK:NO	MULT:YES	CHOOSE:SKILL|Knowledge (History)|Knowledge (Religion)																																											BONUS:SKILL|Knowledge (History),Knowledge (Religion)|1|TYPE=Trait	CSKILL:LIST						SOURCEPAGE:p.9	ASPECT:SaveBonus|+1 trait bonus vs. divine magic.
+Foreign Opportunist			KEY:Trait ~ Foreign Opportunist							CATEGORY:Special Ability	TYPE:Trait.CampaignTrait	PREMULT:1,[PREABILITY:1,CATEGORY=Special Ability,Trait ~ Foreign Opportunist],[PREVAREQ:BypassTraitRestriction,1],[!PREABILITY:1,CATEGORY=Special Ability,TYPE.CampaignTrait]		DESC:You're not a native Osirian, but the opportunity to explore the tombs of Ancient Osirion-and "liberate" the treasures they hold-is too good to pass up. Whether or not you're interested in the history of this land, you're definitely interested in the wealth that's lain hidden in dusty crypts for millennia�such as the tombs in the newly opened necropolis of Wati. You gain a +2 trait bonus on Appraise checks, and that skill is always a class skill for you. In addition, your contacts in the antiquities markets allow you to sell relics of Ancient Osirion for 60% of their listed price, rather than the normal 50% value. What is considered a relic of Ancient Osirion is left to the GM's discretion.																																																																				BONUS:SKILL|Appraise|2|TYPE=Trait																	SOURCEPAGE:p.9
```

Comment:
Regression, description with hyphen spaced now has no spaces and bad character and breaks escaped characters. %% becomes % which will causes errors.